cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,613 @@
+    def is_displayed(self):
+        return self.in_migration_plan
+
+
+class NotStartedPlansView(MigrationPlanView):
+    plans_not_started_list = MigrationPlansList("plans-not-started-list")
+
+    @property
+    def is_displayed(self):
+        expected_title = 'Not Started'
+        on_plan_page = self.in_migration_plan and self.plans_not_started_list.is_displayed
+
+        if self.context['obj'].appliance.version >= '5.10':
+            return on_plan_page and self.title.text == expected_title
+        else:
+            return on_plan_page
+
+
+class InProgressPlansView(MigrationPlanView):
+    # TODO - fix this once dev fixes the locator
+    progress_card = MigrationProgressBar(locator=".//div[3]/div/div[3]/div[3]/div/div")
+
+    @property
+    def is_displayed(self):
+        expected_title = 'In Progress'
+        on_plan_page = self.in_migration_plan and self.progress_card.is_displayed
+
+        if self.context['obj'].appliance.version >= '5.10':
+            return on_plan_page and self.title.text == expected_title
+        else:
+            return on_plan_page
+
+
+class CompletedPlansView(MigrationPlanView):
+    plans_completed_list = MigrationPlansList("plans-complete-list")
+
+    @property
+    def is_displayed(self):
+        expected_title = 'Completed'
+        on_plan_page = self.in_migration_plan and self.plans_completed_list.is_displayed
+
+        if self.context['obj'].appliance.version >= '5.10':
+            return on_plan_page and self.title.text == expected_title
+        else:
+            return on_plan_page
+
+
+class ArchivedPlansView(MigrationPlanView):
+    archived_plans_list = MigrationPlansList("plans-archived-list")
+
+    @property
+    def is_displayed(self):
+        expected_title = 'Archived'
+        on_plan_page = self.in_migration_plan and self.archived_plans_list.is_displayed
+
+        if self.context['obj'].appliance.version >= '5.10':
+            return on_plan_page and self.title.text == expected_title
+        else:
+            return on_plan_page
+
+
+class AddMigrationPlanView(View):
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    back_btn = Button("Back")
+    next_btn = Button("Next")
+    cancel_btn = Button("Cancel")
+    form_title_text = VersionPicker(
+        {Version.lowest(): "Migration Plan Wizard", "5.10": "Create Migration Plan"}
+    )
+
+    @View.nested
+    class general(View):  # noqa
+        infra_map = BootstrapSelect("infrastructure_mapping")
+        name = TextInput(name="name")
+        name_help_text = Text(locator='.//div[contains(@id,"name")]/span')
+        description = TextInput(name="description")
+        select_vm = RadioGroup('.//div[contains(@id,"vm_choice_radio")]')
+
+        @property
+        def is_displayed(self):
+            return self.infra_map.is_displayed and self.name.is_displayed
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button("Import")
+        importcsv_btn = Button("Import CSV")
```

Comment:
I am only seeing one button on this page, am I missing something?

Reply:
Removed